std/socket.d

```diff
@@ -971,7 +971,23 @@ AddressInfo[] getAddressInfo(T...)(in char[] node, T options) @trusted
             static assert(0, "Unknown getAddressInfo option type: " ~ typeof(option).stringof);
     }
 
-    return getAddressInfoImpl(node, service, &hints);
+    return () @trusted { return getAddressInfoImpl(node, service, &hints); }();
```

Comment:
Nevermind, the `hints` parameter has possible unsafe configurations, such as invalid `ai_addrlen`.